components/calendar/src/iso.rs

```diff
@@ -394,6 +394,7 @@ impl Iso {
         fixed + (date.day.0 as i32)
     }
 
+    #[allow(clippy::unwrap_used)]
```

Comment:
"allow" lints should be at the line level (not the function level), and they should be accompanied by a comment explaining why it is safe to do the unwrap/index.

Reply:
These allows are needed as per the addition of deny clippy list on lib.rs . The strategies on how to handle these should change as part of that, we can try to create issues to fix these and reference on the lines as well. The reason behind that is not having full understanding of the code and it makes it harder to complete this specific issue.

Reply:
Observation : Adding these attributes are experimental on expressions
`error[E0658]: attributes on expressions are experimental`